src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -16,6 +25,30 @@ public static Product addProduct(Product product) {
         return product;
     }
 
+    public static User addUser(User user) {
+        if (isNotNull(user)) {
+            user.setId(++userId);
+            users.add(user);
+        }
+        return user;
+    }
+
+    public static Order addOrder(Order order) {
+        if (isNotNull(order)) {
+            order.setId(++orderId);
+            orders.add(order);
+        }
+        return order;
+    }
+
+    public static ShoppingCart addShopingCart(ShoppingCart shoppingCart) {
+        if (isNotNull(shoppingCart)) {
```

Comment:
we can use standard method `Objects.nonNull()`

Reply:
fixed